xknx/devices/climate.py

```diff
@@ -13,27 +13,78 @@
 from .remote_value_switch import RemoteValueSwitch
 from .remote_value_temp import RemoteValueTemp
 
+DEFAULT_SETPOINT_SHIFT_STEP = 0.5
+DEFAULT_SETPOINT_SHIFT_MAX = 6
+DEFAULT_SETPOINT_SHIFT_MIN = -6
+
+
+class SetpointShiftValue(RemoteValue1Count):
+    """Class for managing setpoint_shift values."""
+
+    def __init__(self,
+                 xknx,
+                 group_address=None,
+                 group_address_state=None,
+                 device_name=None,
+                 after_update_cb=None,
+                 setpoint_shift_step=DEFAULT_SETPOINT_SHIFT_STEP,
+                 min_kelvin=DEFAULT_SETPOINT_SHIFT_MIN,
```

Comment:
-> setpoint_shift.min_kelvin to differentiate between the actual bus value and the human readable value. (Kelvin is commonly used for temperature changes or differences instead of Celsius)